go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/livepeer/go-livepeer
 
 go 1.21.5
 
+replace github.com/livepeer/ai-worker => github.com/j0sh/livepeer-ai-worker v0.0.0-20241017214425-b4f177ea52a7
```

Comment:
I think we need to remove it before merging.

Reply:
Yep, actually first we need https://github.com/livepeer/ai-worker/pull/229 or maybe https://github.com/livepeer/ai-worker/pull/233 first in order to define the pipeline schema